server/src/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,14 @@
+class ReviewSerializer {
+  static getSummary(review) {
+    const allowedAttributes = ['rating', 'title', 'content']
```

Comment:
Probably also want to include the `id` so that when we `map` over this array of reviews we can provide the `id` for the React `map` `key`!